docs/UserGuide.adoc

```diff
@@ -52,110 +49,45 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 * Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 ====
 
-=== Viewing help : `help`
+=== Ingredients inventory commands
```

Comment:
Maybe we can just remove all the commands for now, because we are changing some commands to make them easier to use

Reply:
ok I left them in as a guideline, but I think we can simply check the sample's. reverted.